src/components/Table/Table/__stories__/helpers.js

```diff
@@ -32,16 +82,127 @@ export const emailColumns = [
   {
     id: "sentOn",
     title: "Sent on",
-    width: 150
+    width: 150,
+    contentSkeleton: "long-text"
   },
   {
     id: "subject",
-    title: "Subject"
+    title: "Subject",
+    contentSkeleton: ["long-text", "medium-text"]
```

Comment:
why both are needed? I think it's enough to state `medium-text` and then internally decide on long and medium based on the row index

Reply:
I thought about it, but this allow more flexibility. you prefer the way where if user chooses medium/long we are alternating between them?

Reply:
this was the design guideline